src/ts/components/ArcBase.tsx

```diff
@@ -119,7 +119,18 @@ export class ArcView extends React.Component<ArcProps, ComponentState> {
     }
 
     public componentDidMount() {
-        loadModules(this.props.scriptUri)

```

Comment:
Leave this section unchanged, as I've addressed this issue in 3.2.0 by allowing users to pass loader options into the components through props.

Reply:
Sounds good. I will create another PR in the 3.2.0 branch.